src/st/rwrp/A59C.c

```diff
@@ -197,7 +204,14 @@ s32 func_8018D880(void) {
     return temp_v1;
 }
 
-INCLUDE_ASM("asm/us/st/rwrp/nonmatchings/A59C", func_8018D8BC);
+s32 func_8018D8BC(void) {
```

Comment:
Take a look at https://raw.githubusercontent.com/Xeeynamo/sotn-decomp/gh-duplicates/duplicates.txt to find these duplicate functions. In this case, this function should be named GetDistanceToPlayerY. If any of your functions have names in other overlays, it would be great if you could pull those names into here.